tests/Makefile.am

```diff
@@ -201,52 +202,40 @@ dist_yaml_TESTS =				\
 	yaml/broken_equals_operand.yaml		\
 	yaml/capitalization.yaml		\
 	yaml/capsnocont.yaml			\
-	yaml/capsword_for_back.yaml		\
+	yaml/capsword.yaml			\
 	yaml/case-sensitivity.yaml		\
 	yaml/compbrlAtCursor_with_equals.yaml	\
 	yaml/computer_braille.yaml		\
 	yaml/emphasis.yaml			\
 	yaml/example_test.yaml			\
-	yaml/foo.yaml				\
 	yaml/hyphenation.yaml			\
 	yaml/hyphenation_nocross_harness.yaml	\
 	yaml/inpos_outpos.yaml			\
 	yaml/input-length.yaml			\
-	yaml/issue-238.yaml			\
 	yaml/issue-332.yaml			\
-	yaml/issue-35.yaml			\
-	yaml/issue-400.yaml			\
-	yaml/issue-401.yaml			\
-	yaml/issue-406.yaml			\
-	yaml/issue-412.yaml			\
 	yaml/issue-479.yaml			\
-	yaml/issue-498.yaml			\
-	yaml/issue-537.yaml			\
-	yaml/issue-583.yaml			\
-	yaml/issue-584.yaml			\
 	yaml/issue-615.yaml			\
-	yaml/issue-77.yaml			\
-	yaml/issue-886.yaml			\
 	yaml/letterDefTest_harness.yaml		\
 	yaml/multipass-backward.yaml		\
 	yaml/multipass-forward.yaml		\
 	yaml/multipass.yaml			\
 	yaml/multipass-negation.yaml		\
 	yaml/multipass-vs-match.yaml		\
-	yaml/new_emph.yaml			\
```

Comment:
If you remove this from `dist_yaml_TESTS` the test will not be run but afaik it will also not be packaged and distributed. I think it would be better to either mark the whole test as an expected failure or just run it (if it passes) and mention the fact that it isn't very useful in the test itself as you already did

Reply:
Hmm, OK, fine by me to just run it (because it passes). But on the other hand, why does it need to be distributed if it is not run?

Reply:
Well, I guess I'm a bit nerdy when it comes to Makefiles. I guess I've been burned too many times with tables missing from the Makefile. 

I think we should distribute what we have in github, so we should include the test files even if they are useless. If they are _really useless_ then we should probably remove them from the repo as well.

Reply:
Fine.

Reply:
Done